internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -196,7 +196,7 @@ func parseUseStatement(statement string) (string, string, error) {
 	statement = removeStatementTerminator(statement)
 	words := strings.Fields(statement)
 	if len(words) < 2 {
-		return "", "", &types.StatementError{Msg: "Error: Missing database/catalog name: Usage examples: USE DB1 OR USE CATALOG METADATA."}
+		return "", "", &types.StatementError{Msg: "Missing database/catalog name: Usage examples: USE DB1 OR USE CATALOG METADATA."}
```

Comment:
The "usage example(s)" string is a bit inconsistent. Can we introduce a `Usage []string` field here so we can format usage in a consistent way?

Reply:
Done!